lang/java/reef-runtime-local/src/main/java/org/apache/reef/runtime/local/driver/ContainerManager.java

```diff
@@ -207,12 +211,23 @@ private void sendNodeDescriptors() {
         final String id = idmaker.getNextID();
         this.racksPerNode.put(id, rackName);
         this.freeNodesPerRack.get(rackName).put(id, Boolean.TRUE);
+
+        int memorySizeInMB;
```

Comment:
This can be made `final`. If not, move this code to a method of its own to allow it to be `final` here.